python/cucim/pyproject.toml

```diff
@@ -39,8 +39,8 @@ classifiers = [
     "Topic :: Scientific/Engineering",
     "Operating System :: POSIX :: Linux",
     "Environment :: Console",
-    "Environment :: GPU :: NVIDIA CUDA :: 11.0",
-    "Environment :: GPU :: NVIDIA CUDA :: 12.0",
+    "Environment :: GPU :: NVIDIA CUDA :: 11",
+    "Environment :: GPU :: NVIDIA CUDA :: 12",
     "License :: OSI Approved :: Apache Software License",
     "Programming Language :: C++",
     "Programming Language :: Python",
```

Comment:
Should we change from
```
    "Programming Language :: Python :: 3",
    "Programming Language :: Python :: 3.8",
    "Programming Language :: Python :: 3.9",
    "Programming Language :: Python :: 3.10",
    "Programming Language :: Python :: 3.11",
```

to just
```
    "Programming Language :: Python :: 3",
    "Programming Language :: Python :: 3.8",
    "Programming Language :: Python :: 3.10",
```

Reply:
Pushed a change. Though instead of listing minor versions (as the wheels are already Python version specific), just changed to 3. Please let me know what you think